.changes/591.json

```diff
@@ -9,5 +9,10 @@
         "description": "remove Linux image from `mips-unknown-linux-gnu`.",
         "type": "removed",
         "breaking": true
+    },
+    {
+        "type": "changed",
+        "description": "change powerpc64, riscv64, and sparc64 *-linux-gnu images to use a 6.x kernel instead of a 5.x kernel.",
```

Comment:
they went from 4* to 5* in this bump, no?

Reply:
Done, I've consolidated the two and changed the changelog to be something like:

```json
[
    {
        "type": "changed",
        "description": "upgraded most linux images to use a 5.x kernel instead of a 4.x kernel.",
        "breaking": true
    }
    {
        "type": "changed",
        "description": "updated powerpc64, riscv64, and sparc64 *-linux-gnu images to use a 6.x kernel instead of a 4.x kernel.",
        "breaking": true,
        "issues": [1094]
    }
]
```